code/modules/admin/admin.dm

```diff
@@ -2967,6 +2967,42 @@ var/global/noir = 0
 							tgui_alert(usr,"You must be at least a Primary Administrator to affect player reagents.")
 							return
 
+					if ("animate_one")
+						if (src.level >= LEVEL_PA)
+							var/mob/M = input("Which mob?","Adding animation") as null|mob in world
+							if (!M)
+								return
+
+							var/animationpick = tgui_input_list(usr, "Select animation.", "Animation", global.animations)
+							if (!animationpick)
+								return
+							call(animationpick)(M)
+
+							message_admins("[key_name(usr)] added animation [animationpick] to [M].")
+							logTheThing(LOG_ADMIN, usr, "added animation [animationpick] to [M].")
+							logTheThing(LOG_DIARY, usr, "added animation [animationpick] to [M].", "admin")
+						else
+							tgui_alert(usr,"You must be at least a Primary Administrator to affect player reagents.")
```

Comment:
mmm reagents

Reply:
shameless copy-paster EXPOSED in real time